dcos_tunnel/cli.py

```diff
@@ -611,11 +620,26 @@ def _vpn(port, config_file, user, privileged, ssh_port, host,
         clientconfigfile, clientconfigpath = config_tup
         clientfile, clientpath = client_tup
 
+        try_sudo = False
+        if not docker_cmd:
+            docker_cmd = 'docker'
+            try_sudo = True
+        scom = '{} version'.format(docker_cmd)
+        try:
+            ssh_exec_fatal(client, scom)
+        except DCOSException:
+            if try_sudo:
+                docker_cmd = 'sudo ' + docker_cmd
+                scom = '{} version'.format(docker_cmd)
+                ssh_exec_fatal(client, scom)
+            else:
+                raise
```

Comment:
Ah actually this one is correct, there should be a try/except around the `ssh_exec_fatal` in the `try_sudo` "if statement" instead that has the more informative error message.